PluralKit.Bot/Commands/CommandTree.cs

```diff
@@ -307,6 +307,13 @@ private async Task HandleSystemCommand(Context ctx)
                 await PrintCommandList(ctx, "systems", SystemCommands);
             else if (ctx.Match("groups", "gs", "g"))
                 await ctx.Execute<Groups>(GroupList, g => g.ListSystemGroups(ctx, null));
+            else if (ctx.Match("group"))
+                if (ctx.Match("list", "l"))
```

Comment:
This just does nothing if someone types `pk;s group`, not great. why not just add `group` to the match above that?